### PR TITLE
feat: Display error to user when SSHing into offline workspace

### PR DESCRIPTION
**Pull request by kylecarbs**
Very fun lil hack to display user output on an SSH connection to offline workspaces.

image

Here's what it looks like from VS Code:

image

### CODE REVIEW

**Comment by kylecarbs** [comment]
[ch15894]

File: internal/cmd/tunnel.go [hunk 1]
```diff
@@ -59,20 +62,34 @@ coder tunnel my-dev 3000 3000
...
 
-			var workspaceID string
-			for _, workspace := range workspaces {
-				if workspace.Name == args[0] {
-					workspaceID = workspace.ID
-					break
+			if workspace.LatestStat.ContainerStatus != coder.WorkspaceOn {
+				color.NoColor = false
+				notAvailableError := clog.Error("workspace not available",
+					fmt.Sprintf("current status: \"%s\"", workspace.LatestStat.ContainerStatus),
```
Comment (jawnsy) [inline internal/cmd/tunnel.go:73]:
```suggestion
					fmt.Sprintf("current status: %q", workspace.LatestStat.ContainerStatus),
```

:stuck_out_tongue:

**Review by jawnsy — APPROVED** [review]
nice! what was the error message before?

File: internal/cmd/tunnel.go (hunk 1 above)
Comment (kylecarbs) [reply internal/cmd/tunnel.go]: hahah ty

**Comment by kylecarbs** [comment]
A context deadline exceeded :face_with_head_bandage: It'd attempt to connect and exchange candidates but since a workspace wasn't listening nothing would happen!

**Pull request merged**